docs/proposed/psa-driver-interface.md

```diff
@@ -321,6 +321,81 @@ TODO: key input and output for opaque drivers; deterministic key generation for
 
 TODO
 
+#### PAKE operation driver dispatch logic
+
+PSA PAKE operation structure for driver dispatch:
+
+```
+struct psa_pake_operation_s
```

Comment:
I think it's good to describe the general approach taken for driver dispatch here, but I think that this specific structure prototype can be left out since it belongs to the core.